bin/node.py

```diff
@@ -10,8 +11,18 @@ def main():
     rospy.loginfo('sim_adapter_node is initialized')
     simulator = Simulator()
 
-    # TODO: set different adapter based on argv
-    simulator.set_adapter(DiRASimAdapter(simulator))
+    adapter = rospy.get_param('~adapter', default='dira')
+
+    rospy.loginfo(f'Choose adapter for {adapter} simulation')
+
+    if adapter not in ['dira', 'uit']:
+        raise ValueError(f'Adapter must be in [dira, uit]')
+
+    if adapter == 'dira':
+        simulator.set_adapter(DiRASimAdapter(simulator))
+    if adapter == 'uit':
+        simulator.set_adapter(UITSimAdapter(simulator))
+
```

Comment:
I've add parameter to select adapters.

Reply:
It looks nice, however, please update the logging as I mentioned above :+1:

Reply:
Ok I've updated to C-style.